### PR TITLE
Update fleets in Getting started guides

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/getting-started.md [hunk 1]
```diff
@@ -63,21 +63,20 @@ Clicking *Advanced* presents the option to select the rate at which your device
 
 <img alt="Advanced options" src="/img/common/app/advanced.png" width="80%">
 
-Once you have finished your image configuration, click the *Download {{ $names.os.lower }}* button. When the download completes, you should have a zipped image file with a name like `{{ $names.company.short }}-First-App-2.47.1+rev1-v10.6.27.img.zip`, where `First-App` is the name you gave your application on the dashboard.
+Once you have finished your image configuration, click the *Download {{ $names.os.lower }}* button. When the download completes, you should have a zipped image file with a name like `{{ $names.company.short }}-First-App-2.47.1+rev1-v10.6.27.img.zip`, where `First-App` is the name you gave your fleet on the dashboard.
```
Comment (chrisys) [inline pages/learn/getting-started.md:66]:
```suggestion
Once you have finished your image configuration, click the *Download {{ $names.os.lower }}* button. When the download completes, you should have a zipped image file with a name like `{{ $names.company.short }}-First-Fleet-2.47.1+rev1-v10.6.27.img.zip`, where `First-Fleet` is the name you gave your fleet on the dashboard.
```

File: pages/learn/getting-started.md [hunk 2]
```diff
@@ -108,7 +107,7 @@ authorize the CLI, click the _Authorize_ button and head back to the terminal.
 
 __Note:__ Other authentication methods include using your username and password credentials or obtaining an [authentication token][token] from the dashboard. Authentication tokens come in two types, API tokens, and JSON Web Token (JWT) session tokens. While API tokens do not expire, JWT session tokens do after 7 days.
 
-After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} apps` command, which should return information about the application you created in the previous step. Take a note of the `APP NAME` as you'll need this in the next step to push the code to all devices in that application.
+After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} apps` command, which should return information about the fleet you created in the previous step. Take a note of the `APP NAME` as you'll need this in the next step to push the code to all devices in that fleet.
```
Comment (chrisys) [inline pages/learn/getting-started.md:110]:
```suggestion
After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} fleets` command, which should return information about the fleet you created in the previous step. Take a note of the `FLEET NAME` as you'll need this in the next step to push the code to all devices in that fleet.
```

File: pages/learn/getting-started.md [hunk 3]
```diff
@@ -108,7 +107,7 @@ authorize the CLI, click the _Authorize_ button and head back to the terminal.
...
 
 ```shell
 $ {{ $names.company.lower }} apps
```
Comment (chrisys) [inline pages/learn/getting-started.md:113]:
```suggestion
$ {{ $names.company.lower }} fleets
```

File: pages/learn/getting-started.md [hunk 4]
```diff
@@ -123,13 +122,13 @@ __Note:__ See all the commands available with {{ $names.company.lower }} CLI by
 
 __Note:__ You may also use git to deploy code to a device. If you wish to deploy via git, see the instructions [here][gitdocs].
 
-Now to deploy this code to all device(s) in the application, use the `{{ $names.company.lower }} push First-App` command replacing `First-App` with the name of your application. Ensure you are in the root of the project directory before issuing this command or specify the `--source` option to provide an alternate location of the project directory.
+Now to deploy this code to all device(s) in the fleet, use the `{{ $names.company.lower }} push First-App` command replacing `First-App` with the name of your fleet. Ensure you are in the root of the project directory before issuing this command or specify the `--source` option to provide an alternate location of the project directory.
```
Comment (chrisys) [inline pages/learn/getting-started.md:125]:
```suggestion
Now to deploy this code to all device(s) in the fleet, use the `{{ $names.company.lower }} push First-Fleet` command replacing `First-Fleet` with the name of your fleet. Ensure you are in the root of the project directory before issuing this command or specify the `--source` option to provide an alternate location of the project directory.
```

File: pages/learn/getting-started.md [hunk 5]
```diff
@@ -123,13 +122,13 @@ __Note:__ See all the commands available with {{ $names.company.lower }} CLI by
...
 
 ```shell
 $ {{ $names.company.lower }} push First-App
```
Comment (chrisys) [inline pages/learn/getting-started.md:128]:
```suggestion
$ {{ $names.company.lower }} push First-Fleet
```

File: pages/learn/getting-started.md [hunk 6]
```diff
@@ -170,7 +169,7 @@ You'll know your code has been successfully compiled and built when our friendly
 			             <'
 ```
 
-Your application will then be downloaded and executed by all the devices you have connected in your application fleet. The first push is slower to deploy, but all subsequent pushes are much faster due to [Docker layer sharing][dockerLayerDocs]. You can see the progress of the device code updates on the device dashboard:
+Your application will then be downloaded and executed by all the devices you have connected in your fleet. The first push is slower to deploy, but all subsequent pushes are much faster due to [Docker layer sharing][dockerLayerDocs]. You can see the progress of the device code updates on the device dashboard:
```
Comment (chrisys) [inline pages/learn/getting-started.md:172]:
```suggestion
Your release will then be downloaded and started by all the devices you have connected in your fleet. The first push is slower to deploy, but all subsequent pushes are much faster due to [Docker layer sharing][dockerLayerDocs]. You can see the progress of the device code updates on the device dashboard:
```

File: shared/getting-started/projectSpecific/_default.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-To give your device a public URL, access the device page, and choose the _Public Device URL_ toggle. You may also activate this for many devices in your fleet at the same time via the _Actions_ menu in the Applications dashboard.
+To give your device a public URL, access the device page, and choose the _Public Device URL_ toggle. You may also activate this for many devices in your fleet at the same time via the _Actions_ menu in the fleets dashboard.
```
Comment (chrisys) [inline shared/getting-started/projectSpecific/_default.md:1]:
```suggestion
To give your device a public URL, access the device page, and choose the _Public Device URL_ toggle. You may also activate this for many devices in your fleet at the same time via the _Actions_ menu on the device list within a fleet.
```

File: pages/learn/getting-started.md [hunk 8]
```diff
@@ -108,10 +107,10 @@ authorize the CLI, click the _Authorize_ button and head back to the terminal.
...
 __Note:__ Other authentication methods include using your username and password credentials or obtaining an [authentication token][token] from the dashboard. Authentication tokens come in two types, API tokens, and JSON Web Token (JWT) session tokens. While API tokens do not expire, JWT session tokens do after 7 days.
 
-After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} apps` command, which should return information about the application you created in the previous step. Take a note of the `APP NAME` as you'll need this in the next step to push the code to all devices in that application.
+After logging in, test out the {{ $names.company.lower }} CLI by running the `{{ $names.company.lower }} fleets` command, which should return information about the fleet you created in the previous step. Take a note of the `FLEET NAME` as you'll need this in the next step to push the code to all devices in that fleet.
 
 ```shell
-$ {{ $names.company.lower }} apps
+$ {{ $names.company.lower }} fleets
 ID    APP NAME   DEVICE TYPE          ONLINE DEVICES DEVICE COUNT
 98264 First-App  {{ $device.name }}   0              0
```
Comment (vipulgupta2048) [inline pages/learn/getting-started.md:115]:
```suggestion
ID    FLEET NAME   DEVICE TYPE          ONLINE DEVICES DEVICE COUNT
98264 First-Fleet  {{ $device.name }}   0              0
```

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**